time LatestResultStamp
time CurAnalyzeStamp
int32 FaceGesture
int32 RectFace_x
int32 RectFace_y
int32 RectFace_w
int32 RectFace_h

int32 NONE = 0
int32 Face = 1
